core/templates/dev/head/pages/about.html

```diff
@@ -219,32 +108,314 @@ <h3 style="margin-top: 40px;">Contacting Us</h3>
         Piotr Mitros, Rachel Chen, Rahim Nathwani, Robyn Choo, Tricia Ngoon,
         Vikrant Nanda and Vinamrata Singal.
       </p>
-    </md-card>
-  </div>
+      <div class="create-three-cols-container">
+        <div class="create-three-cols-layout">
+          <div class="letter-groups">
+            <span>A</span>
+              <ul>
```

Comment:
deindent this and the next line by 2. The ul is a sibling of the span, not a descendant.